bot.py

```diff
@@ -1377,13 +1377,12 @@ async def ban(ctx, member:discord.User=None, reason=None, *args):
 
 @bot.command()
 @commands.check(isStaff)
-async def unban(ctx, id:int=0):
+async def unban(ctx, member:discord.User=None):
```

Comment:
How are you planning to use this? If the user is banned, then they aren't a member of the server.

Reply:
If a player gets banned, there @ can still be there. Unless you like ctrl+R your client, you might still have that person's @ still cached.
Otherwise, you can instead put the username#discrim (e.g. `!unban JarJarBinks#1337`) and it should work as intended.

Reply:
Sounds good, thanks for the help